search/mteb/dbpedia/operations/default.json

```diff
@@ -0,0 +1,52 @@
+{
+   "name": "default-text-search",
+   "operation-type": "search",
+   "param-source": "query_param_source",
+   "query_source": "queries.json",
+   "size": 100,
+   "title_field": "title",
+   "title_boost": 10,
+   "text_field": "text",
+   "text_boost": 1,
+   "track_total_hits": false,
+   "include-in-reporting": true
+},
+{
+   "name": "english-analyzed-text-search",
+   "operation-type": "search",
+   "param-source": "query_param_source",
+   "query_source": "queries.json",
+   "size": 100,
+   "title_field": "english_analyzed_title",
+   "title_boost": 10,
+   "text_field": "english_analyzed_text",
+   "text_boost": 1,
+   "track_total_hits": false,
+   "include-in-reporting": true
+},
+{
+   "name": "default-text-search-relevance",
+   "operation-type": "text_search_relevance",
+   "param-source": "relevance_param_source",
+   "query_source": "queries.json",
+   "qrels_source": "test.tsv",
```

Comment:
Do we need the `dev.tsv` file or should it be removed?

Reply:
I will remove it.